contracts/contracts/harvest/Harvester.sol

```diff
@@ -355,12 +373,6 @@ contract Harvester is Governable {
         }
     }
 
-    /**
-     * @dev Swap a reward token for stablecoins on Uniswap. The token must have
-     *       a registered price feed with the price provider.
-     * @param _swapToken Address of the token to swap.
-     * @param _rewardTo Address where to send the share of harvest rewards to
-     */
```

Comment:
We still need this comment. We can just update it to explain the v3 vs v2 functionality.

Reply:
I just moved comments to below _swapV2 and _swapV3 functions.
But if you want, I'll put back to _swap function.
So should I remove the comments for _swapV2 and _swapV3 functions? or keep all 3 parts?

Reply:
All 3 should have comments. They all do something different so they should be updated accordingly.